tests/sentry/snuba/metrics/test_snql.py

```diff
@@ -36,7 +36,7 @@ def setUp(self):
         self.org_id = 666
         self.metric_ids = [0, 1, 2]
         indexer.bulk_record(
-            use_case_id=UseCaseKey.RELEASE_HEALTH,
+            use_case_id=UseCaseKey.PERFORMANCE,
```

Comment:
This use case is release health as your line https://github.com/getsentry/sentry/pull/36308/files#diff-cd6ed2f06582566553d69e6c5ab1136505be2633563bf27e63eb2f84ad047c05R79 says

Reply:
okey, but i see different metrics in bulk_create -> https://github.com/getsentry/sentry/blob/d23fb690138b93944c9a410a3ba6b5e2ab7005e2/tests/sentry/snuba/metrics/test_snql.py#L49-L56

Reply:
Can we maybe separate the this `bulk_record` ? As an example 
```
    indexer.bulk_record(
            use_case_id=UseCaseKey.RELEASE_HEALTH,
            org_strings={
                self.org_id: [
                    "abnormal",
                    "crashed",
                    "errored_preaggr",
                    "errored",
                    "exited",
                    "init",
                    "session.status",
                ]
            },
        )
     indexer.bulk_record(
            use_case_id=UseCaseKey.PERFORMANCE,
            org_strings={
                self.org_id: [
                    TransactionSatisfactionTagValue.FRUSTRATED.value,
                    TransactionSatisfactionTagValue.SATISFIED.value,
                    TransactionSatisfactionTagValue.TOLERATED.value,
                    TransactionStatusTagValue.CANCELLED.value,
                    TransactionStatusTagValue.OK.value,
                    TransactionStatusTagValue.UNKNOWN.value,
                    TransactionTagsKey.TRANSACTION_SATISFACTION.value,
                    TransactionTagsKey.TRANSACTION_STATUS.value,
                ]
            },
        )
```